SmthJedi/license.txt

```diff
@@ -1 +1,2 @@
-These files are free to use, modify, and redistribute without attribution or any other restrictions.
\ No newline at end of file
+These files are free to use, modify, and redistribute without attribution or any other restrictions.
```

Comment:
Should probably add that the original files are made by Thexter.

Reply:
added to all 4 fonts.